p/progressbar/progressbar_ubi_8.4.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : progressbar
+# Version       : v1.0.0
+# Source repo   : https://github.com/schollz/progressbar
+# Tested on     : RHEL ubi 8.4
```

Comment:
please mention either RHEL or UBI or if it's been tested on both specify the same as a comma separated list "RHEL 8.4, UBI 8.4" something like that